skeleton-typescript-webpack/src/_references.ts

```diff
@@ -0,0 +1,21 @@
+﻿// this only contains references to aurelia
+// the other typings are in the typings folder
+
+/// <reference path="../node_modules/aurelia-framework/dist/aurelia-framework.d.ts" />
+/// <reference path="../node_modules/aurelia-logging/dist/aurelia-logging.d.ts" />
+/// <reference path="../node_modules/aurelia-path/dist/aurelia-path.d.ts" />
+/// <reference path="../node_modules/aurelia-dependency-injection/dist/aurelia-dependency-injection.d.ts" />
+/// <reference path="../node_modules/aurelia-loader/dist/aurelia-loader.d.ts" />
+/// <reference path="../node_modules/aurelia-pal/dist/aurelia-pal.d.ts" />
+/// <reference path="../node_modules/aurelia-templating/dist/aurelia-templating.d.ts" />
+/// <reference path="../node_modules/aurelia-metadata/dist/aurelia-metadata.d.ts" />
+/// <reference path="../node_modules/aurelia-fetch-client/dist/aurelia-fetch-client.d.ts" />
+/// <reference path="../node_modules/aurelia-binding/dist/aurelia-binding.d.ts" />
+/// <reference path="../node_modules/aurelia-task-queue/dist/aurelia-task-queue.d.ts" />
+/// <reference path="../node_modules/aurelia-pal-browser/dist/aurelia-pal-browser.d.ts" />
+/// <reference path="../node_modules/aurelia-loader-webpack/dist/aurelia-loader-webpack.d.ts" />
+/// <reference path="../node_modules/aurelia-bootstrapper-webpack/dist/aurelia-bootstrapper-webpack.d.ts" />
+/// <reference path="../node_modules/aurelia-router/dist/aurelia-router.d.ts" />
+/// <reference path="../node_modules/aurelia-route-recognizer/dist/aurelia-route-recognizer.d.ts" />
+/// <reference path="../node_modules/aurelia-event-aggregator/dist/aurelia-event-aggregator.d.ts"/>
+/// <reference path="../node_modules/aurelia-history/dist/aurelia-history.d.ts" />
```

Comment:
No, it has nothing to do with old and new version of the definition files. There are no version numbers in the references.
I just could not get it to work with the definitions in the tsconfig.json. 
I will be making an alternative with the typings tool.